platforms/chibios/drivers/audio_dac_additive.c

```diff
@@ -321,9 +326,13 @@ void audio_driver_initialize(void) {
      *
      * this is done here, reaching directly into the stm32 registers since chibios has not implemented BOFF handling yet
      * (see: chibios/os/hal/ports/STM32/todo.txt '- BOFF handling in DACv1.'
+     *
+     * This Explict setting to 0 does not seem to be needed for the G4 Series
      */
+#if !defined(AUDIO_NO_BOFF)
     DACD1.params->dac->CR &= ~DAC_CR_BOFF1;
     DACD2.params->dac->CR &= ~DAC_CR_BOFF2;
+#endif
```

Comment:
Actually the G4 series also has DAC output buffers, but they are controlled by the `MODE1[2:0]` and `MODE2[2:0]` bits in the `DAC_MCR` register.  But apparently the reset value of that register is 0, which corresponds to the state when the output buffers are enabled.

(In fact, I'm not sure whether that code does anything useful on STM32F303 too, because the BOFF bits are actually in the `cr` field of `DACConfig` in that case, so they should be 0 already.)

Reply:
With regards to the G4 that is the conclusion i came to as well. I had that in my first draft of the description but it seemed very wordy and i wanted to keep to the point. 

Looking at the code for the DAC in the hal_dac_lld.c and hal_dac_lld.h i would agree that those bits should be getting cleared in dacStart with the Config in use. I can't test though if removing them would effect the STM32F303 DAC though

Reply:
I like the idea of combining this suggestion with the one lower down about setting up the trigger defines to select the correct values for timer 6/7 by looking for the `QMK_MCU_SERIES_STMX` defines. That would remove the need to manually set any of these defines for most users